client/src/components/requestsTable/index.tsx

```diff
@@ -0,0 +1,388 @@
+          updatedAt: '2022-10-17T08:26:03.553Z',
+        },
+      },
+      {
+        id: 8,
+        brand: 'Volkswagen',
+        model: 'Jetta 2.0L I4',
+        price: 13000,
+        year: 2016,
+        mileage: 57000,
+        quality: 70,
+        isGoodPrice: false,
+        location: 'palestine - ramallah',
+        state: 'on-market',
+        transmission: 'auto',
+        features: [
+          'Power seats',
+          'Automated Parking',
+          'Parking sensors',
+          'Navigation',
+          'Bluetooth',
+          'Digital display',
+          'Paddle shifters',
+          'Apple carplay/ Android',
+          'Rear entertainment screens',
+          'Premium sound system',
+          'Leather Seats',
+          'Push Start',
+          'Keyless entry',
+        ],
+        description: 'This car had an accedent in the left side but the quality of the body generally is good',
+        fuel: 'petrol',
+        createdAt: '2022-10-17T08:26:03.560Z',
+        updatedAt: '2022-10-17T08:26:03.560Z',
+        customerId: 4,
+        customer: {
+          id: 4,
+          fullName: 'lina',
+          email: 'lina@gmail.com',
+          password: '$2a$10$H6ecuHzDjK7IVyqJB2pKi.kPPMBDjQI4swryJNJgK0hh8M8BLpsBi',
+          phoneNumber: '+9705*4637*69',
+          createdAt: '2022-10-17T08:26:03.553Z',
+          updatedAt: '2022-10-17T08:26:03.553Z',
+        },
+      },
+      {
+        id: 9,
+        brand: 'Mitsubishi',
+        model: 'Montero Sport 3.0 L V6',
+        price: 1400,
+        year: 2017,
+        mileage: 161000,
+        quality: 75,
+        isGoodPrice: true,
+        location: 'palestine - gaza',
+        state: 'on-market',
+        transmission: 'auto',
+        features: [
+          'Automated Parking',
+          'Parking sensors',
+          'Blind spot moitor',
+          'Collision warning system',
+          'Cruise Control',
+          'Steering wheel control',
+          'Auto dimming mirror',
+          'Rear entertainment screens',
+        ],
+        description: 'This car had an accedent in the left side but the quality of the body generally is good',
+        fuel: 'petrol',
+        createdAt: '2022-10-17T08:26:03.560Z',
+        updatedAt: '2022-10-17T08:26:03.560Z',
+        customerId: 1,
+        customer: {
+          id: 1,
+          fullName: 'husam',
+          email: 'husam@gmail.com',
+          password: '$2a$10$H6ecuHzDjK7IVyqJB2pKi.kPPMBDjQI4swryJNJgK0hh8M8BLpsBi',
+          phoneNumber: '+9705*46374*9',
+          createdAt: '2022-10-17T08:26:03.553Z',
+          updatedAt: '2022-10-17T08:26:03.553Z',
+        },
+      },
+    ],
+  },
+};
+
+type Props = {
+  status : string
+};
+
+function RequestsTable(props:Props) {
+  const { status } = props;
+  const [carsData, setCarsData] = useState<CarsWithCustomerData >();
+  const [pageCount, setPageCount] = useState<number>(1);
+  const [requestsState, setRequestsState] = useState<string>(status);
```

Comment:
you don`t want requestsState here you can fetch data by status props because here requestsState  doesn`t change